### PR TITLE
Semantic Slots: adding successText and changing errorText, warningText.

### PR DESCRIPTION
**Pull request by aneeshack4**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Adding successText semantic slot & setting it to white in Strong along with changing errorText, warningText. This is being done at the request of Sharepoint team to fix an accessibility bug of theirs.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9489)

### CODE REVIEW

File: packages/styling/src/interfaces/ISemanticTextColors.ts [hunk 1]
```diff
@@ -67,11 +67,17 @@ export interface ISemanticTextColors {
...
    */
   errorText: string;
+
   /**
    * The color of text on errorBackground, warningBackground, blockingBackground, or successBackground.
    */
   warningText: string;
 
+  /**
+   * The color of text on successBackground and bodyBackground.
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticTextColors.ts]: "the color of text indicating [something].... "

File: packages/styling/src/interfaces/ISemanticTextColors.ts [hunk 2]
```diff
@@ -67,11 +67,17 @@ export interface ISemanticTextColors {
    * The default color of error text, used on bodyBackground.
    */
   errorText: string;
+
   /**
    * The color of text on errorBackground, warningBackground, blockingBackground, or successBackground.
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticTextColors.ts]: may want to amend this one too, to say the default color of text on ...

**Comment by KevinTCoughlin** [comment]
@aneeshack4 CI should pass after you commit `npm run update-api`. Change seems reasonable. I'm curious to know more about the bug if you don't mind linking offline.

**Comment by aneeshack4** [comment]
@KevinTCoughlin Yes I've been trying to do that but that command fails with a bunch of errors every time - even after I updated my 6.0 branch with the latest on upstream/6.0 - and the errors are unrelated  to these changes so I'm working on getting this fixed.

**Pull request merged**